README.md

```diff
@@ -502,6 +511,91 @@ server:start()
 
 [socket_ref]: https://www.tarantool.io/en/doc/latest/reference/reference_lua/socket/#socket-tcp-server
 
+## Roles
+
+New roles could be accessed from this project.
+
+### `httpd` role
+
+This role can be used only with Tarantool 3. It allows configuring one or more
+HTTP servers. Those servers could be reused by several other roles.
+
+Example of the configuration:
+
+```yaml
+roles_cfg:
+  roles.httpd:
+    default:
+    - listen: 8081
+    additional:
+    - listen: '127.0.0.1:8082'
+```
+
+Server address should be provided either as a URI or as a single port
+(in this case, `0.0.0.0` address is used).
+
+User can access every working HTTP server from the configuration by name,
+using `get_server(name)` method.
+If the `name` argument is `nil`, the default server is returned
+(its name should be equal to constant `DEFAULT_SERVER_NAME`).
+
+Let's look at the example of using this role. Consider a new role `hello_world`:
+```lua
+local http_client = require('http.client')
+
+local M = {}
+local server = {}
+
+M.apply = function(conf, http_server_role)
+    server = http_server_role.get_server(conf.httpd).httpd
+
+    server:route({
+        path = '/hello/world'
+    }, function(tx)
+        return tx:render({text = 'Hello, world!'})
+    end)
+end
+
+M.get = function()
+    local server_uri = ('http://%s:%s'):format(server.host, server.port)
+    return http_client.get(server_uri .. '/hello/world')
+end
+
+return M
+```
+
+This role accepts a server by name from a config and creates a route to return 
+`Hello, world!` to every request by this route.
+
+Config for `httpd` and `hello_world` might look like this:
+```yaml
+roles_cfg = {
+  httpd = {
+    default = {
+      listen = 8081,
+    },
+    additional = {
+      listen = '127.0.0.1:8082',
+    }
+  },
+  hello_world = {
+    httpd = 'additional',
+  },
+}
+```
+
+After that, we can get `Hello, world!` by just running this script on a
+Tarantool instance:
+```lua
+local httpd_role = require('roles.httpd')
+local hello_role = require('roles.hello_world')
+
+httpd_role.apply(config.roles_cfg.httpd)
+hello_role.apply(config.roles_cfg.hello_world, httpd_role)
+
+print(hello_role.get().body)
+```
```

Comment:
Nobody starts Tarantool like that, we do it in such way for testing. It's better to replace the example with a real cluster configuration file + `tt start`. See examples in Tarantool's docs: https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets .